breathecode/events/tests/urls/test_academy_eventbrite_webhook.py

```diff
@@ -5,24 +5,95 @@
 from breathecode.utils import Cache
 from unittest.mock import patch
 from ..mixins.new_events_tests_case import EventTestCase
+from django.utils import timezone
```

Comment:
Bad test name, this need to have any relation with the path of endpoint or the name of it